### ISSUE TITLE
Some custom module break Settings admin item menu localization

### ISSUE THREAD

**Issue by GioviQ** [OrchardCMS/Orchard#7224]
![modifica blog post quarella net](https://cloud.githubusercontent.com/assets/7103057/19019652/064ad944-888f-11e6-8bae-a695e36bfe60.png)

The issue is documented here https://github.com/JadeX/Orchard.HtmlMinifier/issues/2

But the solution to change the namespace seems strange to me. I use other modules that do not break localization and do not use  Orchard.Core.Settings namespace.
What is the real problem?

**Comment by rtpHarry** [comment]
I dont think you were in the triage meeting when we looked at this one and I see we didn't comment.

Basically @sebastienros looked at it and said ohhhh thats a tough one, I don't know how to fix that.

I think the general idea of what he was saying is that multiple modules all contribute to the Settings menu and the last one in the queue will provide the translation for that menu entry. If that module doesn't have a translation in your language for the Settings string then it uses the default English. 

I guess there is no simple way to get around this.
